.istanbul.yml

```diff
@@ -1,6 +1,6 @@
 verbose: false
 instrumentation:
-  excludes: ['coverage/**', 'index.js', 'cz_config.js']
+  excludes: ['coverage/**', 'index.js', 'cz_config.js', 'server/graphQL/**']
```

Comment:
This is because of no tests for GRAPHQL yet ?

Reply:
yes